test/dynamo/test_dynamo.py

```diff
@@ -101,33 +101,43 @@ def __init__(self, device=None):
         super().__init__()
         self.self_tensor = torch.zeros((5, 3), device=device)
 
-      def forward(self, index, copy_tensor, input_tensor):
+      def copy_(self, index, copy_tensor):
         self.self_tensor.index_copy_(0, index, copy_tensor)
+
+      def add_(self, index, other_tensor):
+        self.self_tensor.add_(other_tensor)
+
+      def abs_(self, index, other_tensor):
+        self.self_tensor.abs_()
+
+      def forward(self, index, copy_tensor, input_tensor, op_name):
+        getattr(self, op_name)(index, copy_tensor)
         output = input_tensor + self.self_tensor
         return output
 
     torch._dynamo.reset()
     met.clear_counters()
     met.clear_all()
     device = xm.xla_device()
-    input_tensor = torch.ones(3)
-    copy_tensor = torch.tensor([[1, 2, 3], [4, 5, 6], [7, 8, 9]],
-                               dtype=torch.float)
-    index = torch.tensor([0, 4, 2])
-    xla_input_tensor = input_tensor.to(device)
-    xla_copy_tensor = copy_tensor.to(device)
-    xla_index = index.to(device)
 
     cpu_model = TestModel()
-    res_cpu = cpu_model.forward(index, copy_tensor, input_tensor)
-
     xla_model = TestModel(device).to(device)
     compiled_model = torch.compile(xla_model, backend='openxla')
-    res_xla_dynamo = compiled_model.forward(xla_index, xla_copy_tensor,
-                                            xla_input_tensor)
 
-    self.assertIn('xla::index_copy', met.counter_names())
-    self.assertTrue(torch.allclose(res_cpu, res_xla_dynamo.cpu()))
+    input_tensor = torch.ones(3)
+    copy_tensor = torch.rand(5, 3)
+    index = torch.tensor([0, 4, 2, 1, 3])
+    xla_input_tensor = input_tensor.to(device)
+    xla_copy_tensor = copy_tensor.to(device)
+    xla_index = index.to(device)
+
+    in_place_ops = ['copy_', 'add_', 'abs_']
+    for in_place_op in in_place_ops:
```

Comment:
Updated the test to call multiple in-place ops in iteration, which should trigger separate `extract_compiled_graph` calls.